docs/compiling-mono/linux/index.md

```diff
@@ -15,6 +15,22 @@ Building Mono is a very simple process as Mono has very few external dependencie
 
 On some distribution you will need to install some development tools which are not installed by default.
 
+Compiler/Build dependencies
+---------------------
+
+Mono depends on GCC 4.8+ or Clang 3.3+ (it needs to support C++11) and on CMake 2.8.10.
+
+| Distribution | GCC         | Clang   | CMake      |
+| ------------ | ----------- | ------- | ---------- |
+| Debian 7     | ~~4.7/4.6~~ | ~~3.0~~ | ~~2.8.9~~  |
```

Comment:
not sure I understand what the crossed-out versions mean? Or more generally, what is the purpose of this table?